README.md

```diff
@@ -79,6 +79,16 @@ make your application work, out of the box.
 If you are using Meteor ≥ 1.3, the flag `--server-only` will be used automatically
 to build your application for Meteor mobile integration.
 
+### Memory in the build process
+
+
+By default the node process can only use ~3.7GB of RAM during the build process
+which can'y be too little for large Meteor application, that's why the
```

Comment:
Typo (`can'y`) + I find this paragraph unclear if you can rephrase. Not sure I understand what is prevalent between `TOOL_NODE_FLAGS` and `max-old-space-size`, what can be the value of `TOOL_NODE_FLAGS`, etc.